agent/github_tools.py

```
@@ -1,658 +1,658 @@
+        from packages.auth.rbac import get_user_role
+        store  = get_secrets_store()
+        uid    = _uid(user)
+        role   = get_user_role(user)
+        # Look for a secret tagged "github"
+        recs   = await store.list_for_user(uid, role)
+        for rec in recs:
+            if "github" in rec.tags or rec.name.lower().startswith("github"):
+                value = await store.get_value(rec.secret_id, uid, role)
+                if value:
+                    return value
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)